### ISSUE TITLE
Email composition Body unreadable

### ISSUE THREAD

**Issue by kstev99** [black7375/Firefox-UI-Fix#277]
**Describe the bug**
Using Protonmail (WebMail) in Firefox the input form in the BODY of composed emails is too dark to read.  TO / FROM and SUBJECT are White on Black but the BODY of the email is Black on DK Gray

**Expected behavior**
Email message should be readable

**Screenshots**
![image](https://user-images.githubusercontent.com/32485060/143074091-f1a0845b-182e-4310-97cc-22516179f17e.png)

**Environment:**

 - Distribution
   - [ ] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
   - [ ] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
   - [x] [Lepton's proton style](https://github.com/black7375/Firefox-UI-Fix/tree/proton-style)
 - Firefox Version: 94.0.2
 - OS:
   - [ ] Linux
   - [x] Windows
   - [ ] Mac
   - [ ] Other: 
 - Theme:
   - [ ] Light
   - [x] Dark
   - [ ] Alpenglow (System: Light)
   - [ ] Alpenglow (System: Dark)
   - [x] System Default
   - Also have tried other dark themes. same result
     - GTK Theme Name (Linux only): 
   - [ ] Other: (e.g. [Photon Colors](https://addons.mozilla.org/en-US/firefox/addon/photon-colors/), [Google Chrome Dark](https://addons.mozilla.org/en-US/firefox/addon/google-chrome-dark/), [Fairytale Of Nature](https://addons.mozilla.org/en-US/firefox/addon/fairytale-of-nature/))

**Additional context**
Input Form on standard forms look OK, just not email Body on Composition of email.  Received emails appear OK as well

**Comment by black7375** [comment]
It may have been fixed with a hotfix. https://github.com/black7375/Firefox-UI-Fix/commit/eb43436f489eae6bb1b68eb3a3c0f9374e9ea02e
Check out the latest version!!

![image](https://user-images.githubusercontent.com/25581533/143149828-f48fecf5-35a4-48ed-a02a-0a2694be0583.png)

**Comment by kstev99** [comment]
Thanks Black and I apologize.  The issue was from some code that I had added to usercontent.css that is supposed to prevent that blinding white flash that sometimes occurs when opening new tabs

/* prevent white flickering, when switching pages */
/*tabpanels#tabbrowser-tabpanels {
  background-color: #000 !important; */

It seemed to have worked, but I guess the email composition is a bad side-effect.  I have tried many other fixes as well so I guess I'll see if the white flashes return.  I removed this code above and the email forms look as they should.  Thanks for all you do
